packages/yoastseo/src/scoring/assessments/SCORING SEO.md

```diff
@@ -168,13 +168,14 @@ With the example keyphrase `cat and dog` the following criteria would apply to c
 
 **What is counted as a keyphrase match**: ≥50% of all (content) words from the keyphrase in the alt tag.
 
-| Traffic light   	      | Score	     | Criterion | Feedback |
-|------------	|------------------	|---------------------	|---------------	|
-| Orange (cornerstone: red)		| 6 (cornerstone: 3)		| No images with keyphrase/synonym in alt-tags		| **Image Keyphrase**: Images on this page do not have alt attributes with at least half of the words from your keyphrase. **Fix that!** |
-| Orange	| 6	| There are at least 5 images and less than 30% have an alt-tag with keyphrase/synonym		| **Image Keyphrase**: Out of X images on this page, only X have an alt attribute that reflects the topic of your text. **Add your keyphrase or synonyms to the alt tags of more relevant images!** |
-| Orange	| 6	| There are at least 5 images and more than 75% have an alt-tag with keyphrase/synonym		| **Image Keyphrase**: Out of X images on this page, X have alt attributes with words from your keyphrase or synonyms. That's a bit much. **Only include the keyphrase when it really fits the image**. |
-| Green	| 9	| There are less than 5 images and at least one has an alt-tag with a keyphrase/synonym		| **Image Keyphrase**: Good job! |
-| Green	| 9	| There are at least 5 images and between 30 and 75% have an alt-tag with a keyphrase/synonym		| **Image Keyphrase**: Good job! |
+| Traffic light   	      | Score	     | Criterion                                                                                     | Feedback |
+|------------	|------------------	|-----------------------------------------------------------------------------------------------|---------------	|
+| Orange (cornerstone: red)		| 6 (cornerstone: 3)		| No images with keyphrase/synonym in alt-tags		                                                | **Image Keyphrase**: Images on this page do not have alt attributes that reflect the topic of your text. **Add your keyphrase or synonyms to the alt tags of relevant images!** |
+| Orange	| 6	| There are at least 5 images and less than 30% have an alt-tag with keyphrase/synonym		        | **Image Keyphrase**: Out of X images on this page, only X have an alt attribute that reflects the topic of your text. **Add your keyphrase or synonyms to the alt tags of more relevant images!** |
+| Orange	| 6	| There are at least 5 images and more than 75% have an alt-tag with keyphrase/synonym		        | **Image Keyphrase**: Out of X images on this page, X have alt attributes with words from your keyphrase or synonyms. That's a bit much. **Only include the keyphrase when it really fits the image**. |
+| Orange	| 6	| There are 5 images and 80% have an alt-tag with keyphrase/synonym		                           | **Image Keyphrase**: Good job! |
```

Comment:
The bullet should be `green`, the score should be `9`. And following the discussion in [the Slack thread](https://yoast.slack.com/archives/C03PRESAXMJ/p1679672136598219?thread_ts=1679665677.706549&cid=C03PRESAXMJ), I think the correct criteria would be: "There are 5 images and 2-4 images have an alt-tag with keyphrase/synonym"